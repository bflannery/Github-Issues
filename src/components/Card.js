import React from 'react';

const Card = ({
    children,
    classPrefix,
    title
}) => {
    return (
        <div className={`${classPrefix}-card card`}>
            <div className={`${classPrefix}-card-header card__header`}>
                <h4 className={`${classPrefix}-card-header-title card__header-title`}>
                    {title}
                </h4>
            </div>
            <div className="app-page-left-column-card-body card__body">
                {children}
            </div>
        </div>
    )
}

export default Card
