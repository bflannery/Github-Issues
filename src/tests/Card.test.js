import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Card from "../components/Card";

describe('Repo Component', () => {
    let props = {}
    beforeEach(() => {
        props = {
            repo: {
                id: 1,
                name: 'Repo 1'
            }
        }

    })
    it('should render stuff', () => {
        const wrapper = shallow(<Card  props={props}/>)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})
