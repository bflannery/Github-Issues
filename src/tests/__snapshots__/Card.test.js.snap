// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repo Component should render stuff 1`] = `
<div
  className="undefined-card card"
>
  <div
    className="undefined-card-header card__header"
  >
    <h4
      className="undefined-card-header-title card__header-title"
    />
  </div>
  <div
    className="app-page-left-column-card-body card__body"
  />
</div>
`;
